src/components/Input/Input.js

```diff
@@ -51,6 +51,13 @@ const inputOptionalStyles = ({ theme, optional }) =>
     box-shadow: none;
   `;
 
+const inputTextAlignRightStyles = ({ textAlign }) =>
+  textAlign === 'right' &&
```

Comment:
I think I would turn this into a boolean prop. After all there is only `left` and `right`, of which one is the default. So a prop like `rightAligned` seems like a good option, no?

Reply:
i think it can have all legal css text-align values, we just use two of them (for now). also the name is already familiar and thats why easier to predict how it works

Reply:
I also actually like that it mirrors the DOM. As ju said, it's predictable. Since we know that alignment can have more than one value (why never a center input? the mobile app has it), let's try and contain our booleans and go for something DOM-like.

Reply:
I agree with Monica and Ju :)